woocommerce/woocommerce-android
# Issue/5124 add track event variation details

Closes: #5124 


### Description


Adds a tracking event to the Attributes tab in product variation detail screen 

### Testing instructions


- Go to settings in the app -> Privacy -> enable tracking
- Go to the Products tab and select a variable product 
- tap on variations
- tap on one of the variations
- tap on Attributes

<img src = "https://user-images.githubusercontent.com/30724184/170482767-5e7baeac-7c4e-477d-8452-a0acc7cb2157.png" width="350" >

- Go to the app logs
- make sure `PRODUCT_VARIATION_DETAILS_ATTRIBUTES_TAPPED` is being tracked

<img src = "https://user-images.githubusercontent.com/30724184/170483199-4ceeabed-d1d1-4114-b217-0bdaf171534e.png" width="350" >


### Images/gif
